pkg/installmanager/fake.go

```diff
@@ -0,0 +1,43 @@
+package installmanager
+
+import (
+	"encoding/json"
+	"fmt"
+
+	installertypes "github.com/openshift/installer/pkg/types"
+	"github.com/pkg/errors"
+
+	hivev1 "github.com/openshift/hive/pkg/apis/hive/v1"
+)
+
+const fakeMetadata = `{"aws":{"identifier":[{"kubernetes.io/cluster/fake-infraid":"owned"},{"openshiftClusterID":"fake-cluster-id"}],"region":"us-east-1"},"clusterID":"fake-cluster-id","clusterName":"%s","infraID":"fake-infra-id"}`
```

Comment:
since this is supposed to be used as format string, i would name is accordingly.